node/utils.ts

```diff
@@ -52,3 +56,35 @@ export const hashString = (str: string) => {
 }
 
 export const getBucket = (prefix: string, bucketName: string) => `${prefix}_${bucketName}`
+
+export const startSitemapGeneration = async (ctx: Context) => {
+  const { clients: { vbase, events }, vtex: { logger, adminUserAuthToken } } = ctx
+  if (!adminUserAuthToken) {
+      ctx.status = 401
+      ctx.body = 'Missing adminUserAuth token'
+      logger.error(`Missing adminUserAuth token`)
+      return
+  }
+  const force = ctx.query.__force !== undefined
```

Comment:
`ctx.query.__force` can be `null`?
```suggestion
  const force = !!ctx.query.__force
```

Reply:
No it will be undefined or a string, but we only want to check it the key is there, the value is not important.